plugins/Diagnostics/Diagnostic/DbOverSSLCheck.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Piwik\Plugins\Diagnostics\Diagnostic;
+
+use Piwik\Common;
+use Piwik\Config;
+use Piwik\Db;
+use Piwik\Translation\Translator;
+
+/**
+ * Check if Piwik is connected with database through ssl.
+ * TODO: link to piwik FAQ into comment
+ */
+class DbOverSSLCheck implements Diagnostic
+{
+    /**
+     * @var Translator
+     */
+    private $translator;
+
+    public function __construct(Translator $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    public function execute()
+    {
+        $use_ssl = Config::getInstance()->database['use_ssl'];
+        if (!$use_ssl) {
+            return array();
+        }
+
+        $label = $this->translator->translate('Installation_SystemCheckDatabaseSSL');
+
+        $cipher = Db::fetchAll("show status like 'Ssl_cipher'");
+        if(!empty($cipher[0]['Value'])) {
+             return array(DiagnosticResult::singleResult($label, DiagnosticResult::STATUS_OK, $this->translator->translate('Installation_SystemCheckDatabaseSSLCipher') . ': ' . $cipher[0]['Value']));
+        }
+
+        //no cipher, not working
+        $comment = sprintf($this->translator->translate('Installation_SystemCheckDatabaseSSLNotWorking'), "use_ssl", "true");
```

Comment:
in config file, we have habit of asking user to set 1 or 0 rather than true or false. Replace `true` with `1`